### PR TITLE
Display related cards for merc abilities

### PR DESCRIPTION
**Pull request by piedpiper358**

### CODE REVIEW

File: core/src/js/components/mercenaries/overlay/teams/mercenaries-team-ability.component.ts [hunk 1]
```diff
@@ -77,6 +79,7 @@ export class MercenariesTeamAbilityComponent {
 		const abilityCard = this.allCards.getCard(value.cardId);
 		this.type = value.type;
 		this.cardId = value.cardId;
+		this.relatedCardIds = abilityCard.relatedCardDbfIds?.map(String) ?? [];
```
Comment (sebastientromp) [inline core/src/js/components/mercenaries/overlay/teams/mercenaries-team-ability.component.ts:82]: I think this won't work, as the relatedCardDbfIds return dbfIds (numbers), and the relatedCards tooltip expects cardIds. So you probably will need to map the dbfId to a standard cardId

Comment (sebastientromp) [reply core/src/js/components/mercenaries/overlay/teams/mercenaries-team-ability.component.ts:82]: `abilityCard.relatedCardDbfIds?.map(dbfId => this.allCards.getCard(dbfId).id) ?? []`

**Comment by sebastientromp** [comment]
Seems to be good right now. Are you able to post a screenshot of it working?

**Comment by sebastientromp** [comment]
Nice!

**Comment by piedpiper358** [comment]
Only it seems to me that the linked cards are quite small - it's hard to read the text on them.
Maybe add a parameter to CardTooltip to inlarge related cards?

**Pull request merged**